invenio_rdm_records/resources/serializers/csl/schema.py

```diff
@@ -110,7 +123,7 @@ def get_isbn(self, obj):
         """Get ISBN."""
         identifiers = obj["metadata"].get("identifiers", [])
         for identifier in identifiers:
-            if identifier["scheme"] == "ISBN":
+            if identifier["scheme"] == "isbn":
```

Comment:
Should we downcase or are we confident it's always the right casing?

Reply:
Lower case is used for basically all the default vocabularies https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/config.py. I guess someone could customize the vocabulary to use upper case...but that seems like an unusual use case.